### PR TITLE
Small doc and crash fix.

### PR DESCRIPTION
**Pull request by nouiz**

### CODE REVIEW

File: theano/compile/debugmode.py [hunk 1]
```diff
@@ -1753,6 +1753,12 @@ def make_all(self, profiler=None, input_storage=None,
             else:
                 thunks_py.append(None)
 
+            if not self.maker.mode.check_c_code and thunks_py[-1] is None:
+                print("Op %s don't have a perform, forcing check of the c code" % node.op)
```
Comment (lamblin) [inline theano/compile/debugmode.py]: This line is too long for flake8.

Comment (lamblin) [inline theano/compile/debugmode.py]: Also, it should probably use `_logger` instead of `print`.

**Comment by lamblin** [comment]
Otherwise, I'm OK with the change.

**Comment by nouiz** [comment]
done.

**Pull request merged**